### ISSUE TITLE
[Leia/Matrix] Ambiguous translation string in resource.language.pl_pl

### ISSUE THREAD

**Issue by eremem** [xbmc/xbmc#18767]
### Describe the bug
Sorry guys for opening a new issue, but I wasn't sure if the following problem fits the #18712. If so, please connect or merge (or whatever applies).

The following polish translation should be updated:
msgctxt "#19061"
msgid "Add timer"
msgstr "Nagrywaj"  **-> "Dodaj zadanie"**

It can be seen in the program's context menu in the EPG view alongside with the entry:
msgctxt "#264"
msgid "Record"
msgstr "Nagrywaj"

As you can see the polish translation for both #264 and #19061 is "Nagrywaj", so when you open the context menu for any TV/radio program in the EPG view, you can see "Nagrywaj" twice.
With the proposed adjustment for #19061 the ambiguity will be removed and the new translation will harmonize with the one for the symmetric command #19060:
msgctxt "#19060"
msgid "Delete timer"
msgstr "Usuń zadanie"

## Expected Behavior
see _Describe the bug_

## Actual Behavior
see _Describe the bug_

## Possible Fix
I suggest to change the polish translation for #19061 as stated above in _Describe the bug_ in order to name the action properly and avoid the current ambiguity.

### To Reproduce
Go to TV/Guide, select any program, open the context menu. You will see 2 commands with the same caption: "Nagrywaj" (#19061) and "Nagrywaj" (#264)

### Debuglog

N/A

### Screenshots 
Here are some links or screenshots to help explain the problem:
![obraz](https://user-images.githubusercontent.com/56274896/98870289-663c5100-2473-11eb-8e55-0cf53a6df5c2.png) 
![obraz](https://user-images.githubusercontent.com/56274896/98870426-92f06880-2473-11eb-8012-9780c16e1d74.png)

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:
N/A

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [x] Linux
 - [ ] OSX
 - [x] Windows
 - [ ] Windows UWP

 - Operating system version/name:  XBian & Windows 10
 - Kodi version: 18.9.0

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by enen92** [comment]
Should translation issues be tracked in the main repo? Maybe something for @gade01

**Comment by gade01** [comment]
@enen92 
Well, we need to decide if we want to do fixes at Github or Weblate. 

They sync both ways (well, they do anyway when I get the final issues fixed at Weblate).

Weblate has a comment and vote system with suggestions, approvals etc.
Maybe it's more safe to do fixes there?

Maybe for now we can do this fix at Github and push it to Weblate. 
And in the future all translation issues are handled at Weblate?

**Comment by enen92** [comment]
I'd say to use weblate (will it supoort user accounts?) instead of GitHub, the same way translation teams have worked in transifex. With the number of open issues we have already I don't think it helps if we increase this number with translation related problems

**Comment by gade01** [comment]
@eremem 
Thanks so much for reporting this.

We are currently working on moving translations to Weblate.
Currently we still have some remaining things to sort out, which is why we have not yet opened fully up for regular users and translators.

Once we open up, you can fix issues like this by registering as a user.
We will keep the issue open until then.

**Comment by eremem** [comment]
@gade01
Thank you for your feedback. It would be great to hear from you when it will be possible to contribute to the translations.

Since I use Leia and Matrix in parallel at the moment, I noticed that the Polish translations for both these versions require a few corrections or, in case of Matrix, additions too.
At the moment I'm forced to patch the affected strings locally after every update, which admittedly doesn't happen very often but still is a little irritating :).

**Comment by gade01** [comment]
@eremem I'll ping you here when things are ready at Weblate.
You are more than welcome to update and check all Polish translations.

**Comment by gade01** [comment]
@eremem 
Fix pending:
https://github.com/xbmc/repo-resources/pull/125/files#diff-fc1d73e5c3b3fef1967ef3e06e4ea610d28e163676da5cba8fbb66da60e33fb5

Please check if this is correct.

**Comment by eremem** [comment]
**@gade01**
The corrected translations look now good to me.

**Comment by gade01** [comment]
The changes and fixes are pushed to the new language addons.

Thanks again for reporting this.
